src/components/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Navbar from "./Navbar"
+
+const Header = (() => {
+    return(
+        <header className="header-container">
+        <h1 className="header">This is the Header</h1>
+        <Navbar />
```

Comment:
Does the Navbar component need to be a child of the Header component? What was the reason for this?

Reply:
Probably not actually!